external-crates/move/crates/move-compiler/src/naming/translate.rs

```diff
@@ -1707,6 +1708,25 @@ fn exp(context: &mut Context, e: Box<E::Exp>) -> Box<N::Exp> {
                             (bloc, help_msg),
                         ));
                     }
+                    // If no abort code is given for the assert, we add in the abort code as the
+                    // bitset-line-number if `CleverAssertions` is set.
+                    if nes.value.len() == 1
+                        && context.env.check_feature(
+                            context.current_package,
+                            FeatureGate::CleverAssertions,
+                            bloc,
+                        )
```

Comment:
Nit: we shouldn't support this if `is_macro.is_none()` IMO -- it's another way to get people to do the right thing.

Reply:
Great spot! Updated + added a test for this